guides/release/routing/linking-between-routes.md

```diff
@@ -182,18 +191,21 @@ To solve this problem, or maybe to cross-link comments from photos other than
 the currently active one, you can pass an array of model objects using the
 `@models` argument and the `{{array}}` helper:
 
-```handlebars {data-filename=app/templates/photos.hbs}
-<h1>Latest Comments</h1>
-
-<ul>
-  {{#each this.latestComments as |comment|}}
-    <li>
-      <LinkTo @route="photos.photo.comment" @models={{array comment.photo comment}}>
-        {{excerpt comment.body}}...
-      </LinkTo>
-    </li>
-  {{/each}}
-</ul>
+```handlebars {data-filename=app/templates/photos.gjs}
+import { LinkTo } from '@ember/routing';
+<template>
+  <h1>Latest Comments</h1>
+  
+  <ul>
+    {{#each this.latestComments as |comment|}}
```

Comment:
Does `this.latestComments` still work in route templates?
Shouldn't this be `@controller.latestComments`?

Reply:
In that case, it probably applies to the other references to `this` as well so those also should be updated

Reply:
Thanks for the feedback! I think I know what to do, I'm working on addressing this now then I'll ask for more feedback.

Reply:
I hate controllers, so I'm a fan of using the stateful component pattern instead of controller lol